tests/rpc_performance_test.ts

```diff
@@ -0,0 +1,64 @@
+import { ApiPromise, WsProvider } from '@polkadot/api';
+import { Keyring } from '@polkadot/keyring';
+import { manta_pay_types, rpc_api } from './types';
+import { StoragePrepareConfig, setup_storage, manta_pay_config} from './manta_pay';
+import minimist, { ParsedArgs } from 'minimist';
+import { performance } from 'perf_hooks';
+import { expect } from 'chai';
+
+const test_config = {
+    ws_address: "ws://127.0.0.1:9800",
```

Comment:
Can the configuration be put in a toml file?

Reply:
Maybe yeah. @Apokalip can look into while implementing the framework.

Reply:
Do you mean like a config toml containing all test configurations? If it's that it should be possible with typescript I think, but I do need to look into it.

Reply:
Sometimes, I start a node with port like `9988`, so I have to modify the port in related source code.

Reply:
Well if we put it in typescript we can have extra arguments to modify the config

Reply:
Typescript tests are meant to be part of CI, not of nodes running locally on a dev PC.
So IMHO, just hardcode it. If you *need* to run the CI tests locally, just use the default ports

Reply:
All the CI tests just pass --address argument to overwrite this.